addon/components/lt-body.js

```diff
@@ -239,6 +290,55 @@ export default Component.extend({
     this.set('useVirtualScrollbar', fixedHeader || fixedFooter);
   },
 
+  didUpdateAttrs({ newAttrs, oldAttrs }) {
+    this._super(...arguments);
+
+    let newScrollTo = get(newAttrs, 'scrollTo.value');
+    let oldScrollTo = get(oldAttrs, 'scrollTo.value');
+    let newScrollToRow = get(newAttrs, 'scrollToRow.value');
+    let oldScrollToRow = get(oldAttrs, 'scrollToRow.value');
+
+    let targetScrollOffset = 0;
+    let hasReachedTargetScrollOffset = true;
+
+    if (oldScrollTo !== newScrollTo) {
+      targetScrollOffset = newScrollTo;
+      if (newScrollTo > 0) {
+        hasReachedTargetScrollOffset = false;
+      }
+    } else if (oldScrollToRow !== newScrollToRow) {
+      if (newScrollToRow instanceof Row) {
+        let rowElement = document.getElementById(newScrollToRow.get('rowId'));
+        if (rowElement instanceof Element) {
+          targetScrollOffset = rowElement.offsetTop;
+        }
+      }
+    }
+
+    this.set('targetScrollOffset', targetScrollOffset);
+    this.set('hasReachedTargetScrollOffset', hasReachedTargetScrollOffset);
+  },
+
+  rowObserver: observer('rows.[]', function() {
+    run.scheduleOnce('afterRender', this, this.checkTargetScrollOffset);
```

Comment:
@buschtoens I suggest adding a private method:

```js
_scheduleCheckTargetScrollOffset() {
  run.cancel(this._checkTargetOffsetTimer); // Cancel existing checks since only one needs to happen (I'm not sure if this is actually needed since we are using scheduleOnce.... but just to be safe?
  this._checkTargetOffsetTimer = run.scheduleOnce('afterRender', this, this.checkTargetScrollOffset);
}

destroy() {
  this._super(...arguments);
  run.cancel(this._checkTargetOffsetTimer);
}
```